config/default/accountapp.yaml

```diff
@@ -0,0 +1,34 @@
+ingress:
+  enabled: true
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "kubernetes.io/ingress.class": "public-ingress"
+    "nginx.ingress.kubernetes.io/hsts": "true"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+
```

Comment:
Each container will generate a cookie to store the access, so it's important that a user always sticks to the same container which is why 
`nginx.ingress.kubernetes.io/affinity: cookie` is needed when having multiple replicas

Reply:
why is it stateful 😢 

fixed..